pyk/src/pyk/kcfg/minimize.py

```diff
@@ -3,20 +3,32 @@
 from functools import reduce
 from typing import TYPE_CHECKING
 
-from ..cterm import CTerm
-from ..utils import not_none, single
+from pyk.cterm import CTerm
+from pyk.cterm.cterm import cterm_match
+from pyk.utils import not_none, single
+
+from .semantics import DefaultSemantics
 
 if TYPE_CHECKING:
     from collections.abc import Callable
 
+    from pyk.kast.outer import KDefinition
+
     from .kcfg import KCFG, NodeIdLike
+    from .semantics import KCFGSemantics
 
 
 class KCFGMinimizer:
     kcfg: KCFG
+    heuristics: KCFGSemantics
```

Comment:
```suggestion
    semantics: KCFGSemantics
```

Call it what it is, for easier use later. Either `kcfg_semantics` or just `semantics`.

Reply:
Changed it to `semantics`.